scale/docs/rest/source_file.rst

```diff
@@ -751,11 +751,33 @@ These services provide access to information about source files that Scale has i
 +--------------------+-------------------+----------+---------------------------------------------------------------------+
+| time_field         | String            | Optional | Indicates the time field(s) that *started* and *ended* will use for |
+|                    |                   |          | time filtering. Valid values are:                                   |
+|                    |                   |          |                                                                     |
+|                    |                   |          | - *last_modified* - last modification of source file meta-data      |
+|                    |                   |          | - *data* - data time of source file (*data_started*, *data_ended*)  |
+|                    |                   |          | - *source* - overall time for all associated source files           |
+|                    |                   |          |              (*source_started*, *source_ended*)                     |
+|                    |                   |          |                                                                     |
+|                    |                   |          | The default value is *last_modified*.                               |
++--------------------+-------------------+----------+---------------------------------------------------------------------+
 | order              | String            | Optional | One or more fields to use when ordering the results.                |
 |                    |                   |          | Duplicate it to multi-sort, (ex: order=file_name&order=created).    |
 |                    |                   |          | Nested objects require a delimiter (ex: order=job_type__name).      |
 |                    |                   |          | Prefix fields with a dash to reverse the sort, (ex: order=-created).|
 +--------------------+-------------------+----------+---------------------------------------------------------------------+
+| job_output         | String            | Optional | Return only products for the given job output.                      |
+|                    |                   |          | Duplicate it to filter by multiple values.                          |
++--------------------+-------------------+----------+---------------------------------------------------------------------+
+| recipe_id          | Integer           | Optional | Return only products produced by the given recipe identifier.       |
+|                    |                   |          | Duplicate it to filter by multiple values.                          |
++--------------------+-------------------+----------+---------------------------------------------------------------------+
+| recipe_job         | String            | Optional | Return only products produced by the given recipe job.              |
+|                    |                   |          | Duplicate it to filter by multiple values.                          |
```

Comment:
It doesn't look like the view code actually allows duplicate recipe_job or job_output values. Applies also to products API.